### PR TITLE
Add possible css only fix for height issue #205

### PR DESCRIPTION
**Pull request by domharrington**
This needs further testing

### CODE REVIEW

**Comment by domharrington** [comment]
You can see this broken here: https://preview.readme.io/?selected=swagger-files%2Fboolean.json
Fixed here: http://localhost:9966/?selected=swagger-files%2Fboolean.json

This may break people's custom stylesheets if they've overridden stuff, but we can notify the ones that we're aware of before we deploy it.

**Comment by erunion** [comment]
@rafegoldberg Thoughts on this fix versus https://github.com/readmeio/api-explorer/pull/205?

**Comment by rafegoldberg** [comment]
damn @kanadgupta, good eye... and yeah, @erunion—this can be fixed with the following one-liner on the `#hub-content:before` pseudo, that is:

```diff
  #hub-content:before {
    content: '';
+   box-sizing: border-box;
    width: 390px;
    height: 100%;
    /* etc. */
  }
```

**Comment by rafegoldberg** [comment]
And to your question @erunion, I'd say doing this in pure CSS is preferable to #205; always best to avoid installing new dependencies or adding layout logics in JS!

File: packages/api-explorer/package-lock.json [hunk 1]
```diff
@@ -2104,7 +2104,7 @@
     },
     "event-stream": {
       "version": "3.3.4",
-      "resolved": "http://registry.npmjs.org/event-stream/-/event-stream-3.3.4.tgz",
```
Comment (kanadgupta) [inline packages/api-explorer/package-lock.json]: We can probably also revert this change as well!

**Comment by mjcuva** [comment]
What's the status of this pr? I enabled splitReference for our project and noticed we have the same issue this fixes 😬 https://docs.readme.com/reference/authentication

**Comment by erunion** [comment]
I'll package a release for this up tomorrow or Thursday.

**Comment by mjcuva** [comment]
Moved to https://github.com/readmeio/api-explorer/pull/377 because of merge hell
